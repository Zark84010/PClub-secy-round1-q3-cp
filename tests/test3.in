20 998 6
95004 16224 43992 105183 14300 22932 70980 19721 19721 19721 19721 19721 19721 30784 12987 19721 12987 35113 17797 5772 
c 10 851
f 11 13
f 4 18
c 3 851
f 19 20
f 8 18
