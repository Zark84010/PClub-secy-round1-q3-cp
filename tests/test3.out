19681558
998
480038
998
